library/CMakeLists.txt

```diff
@@ -684,36 +684,41 @@ if(TINYSPLINE_CXX_AVAILABLE AND TARGET_SUPPORTS_SHARED_LIBS)
 
 		# PHP
 		if(TINYSPLINE_WITH_PHP)
-			find_package(PHP)
+			find_package(PHP COMPONENTS devel)
```

Comment:
For the sake of consistency, I would rename devel to Development as it is in the Java module (`find_package(Java COMPONENTS Development)`)

Reply:
Fixed!